style.css

```diff
@@ -19,12 +19,6 @@
     top: 20px;
     bottom: 5px;
     max-width: 100%;
-
-    /* height: 92px; */
-
-    /* display: flex;
-    align-items: flex-start;
-    flex-shrink: 0; */
```

Comment:
- [ ] Please remove the commented lines of code, because commented code in your code base can be detrimental for several reasons. 
     1. It adds clutter and makes the code harder to read and understand. 
     2. commented code may become outdated or irrelevant over time, leading to confusion. 
     3. it hinders collaboration and maintenance, as developers might spend unnecessary time deciphering commented-out code. Instead, use comments purposefully, such as to explain complex logic, document assumptions or constraints, and provide high-level explanations for non-obvious code. 

Comments should be concise, relevant, and focused on adding value to future developers who will work on the codebase.